tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -41,33 +42,45 @@
 class KatelloCertsCheckTestCase(TestCase):
     """Implements ``katello-certs-check`` tests.
 
-    Depends on presence of custom certificates at path given
-    in robottello.properties file.
+    Depends on presence of scripts and files in
+    tests/foreman/data/
+    CA cert (a.k.a cacert.crt or rootCA.pem) can be used as bundle file.
     """
 
     @classmethod
-    @skip_if_not_set('certs')
     def setUpClass(cls):
-        """Get host name and credentials."""
+        """Get host name, scripts, and create working directory."""
         super(KatelloCertsCheckTestCase, cls).setUpClass()
-        _, cls.ca_bundle_file_name = os.path.split(settings.certs.ca_bundle_file)
-        _, cls.cert_file_name = os.path.split(settings.certs.cert_file)
-        _, cls.key_file_name = os.path.split(settings.certs.key_file)
-        cls.ca_bundle_file = settings.certs.ca_bundle_file
-        cls.cert_file = settings.certs.cert_file
-        cls.key_file = settings.certs.key_file
+        _, cls.sat6_hostname = os.path.split(settings.server.hostname)
+        cls.key_file_name = '{0}/{0}.key'.format(cls.sat6_hostname)
+        cls.cert_file_name = '{0}/{0}.crt'.format(cls.sat6_hostname)
+        cls.ca_bundle_file_name = 'cacert.crt'
         cls.SUCCESS_MSG = "Validation succeeded"
-        # uploads certs to satellite
         upload_file(
-            local_file=settings.certs.ca_bundle_file,
-            remote_file="/tmp/{0}".format(cls.ca_bundle_file_name),
+            local_file=get_data_file('generate-ca.sh'), remote_file="{}".format('generate-ca.sh'),
```

Comment:
Why need to use format here?  cant we use directly? any issues with that?

Reply:
@omkarkhatavkar No, I can change that, and the line above needs changing too. (I was experimenting with using subdirectories under /tmp and that is an artefact of that).
Thanks for looking.

Reply:
done